corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -229,6 +229,10 @@ def closed_transactions(self):
         return filter(lambda action: action.action_type == const.CASE_ACTION_CLOSE, self.actions)
 
     @property
+    def transactions(self):
```

Comment:
what's the deal with this?

if we are adding a read only property to CommCareCase we need to check that no one has made a case property on prod with this name, and we also need to add it to the reserved-words thing in app manager. 

not namespacing case properties on `CommCareCase` 6 years ago is one of the worst and far-reaching design decisions I've made in HQ. :disappointed: